lib/sync.rb

```diff
@@ -145,14 +159,19 @@ def pusher_wss_port
       config[:pusher_wss_port]
     end
 
-    def pusher_encrypted
-      if config[:pusher_encrypted].nil?
+    def encryption_flag
+      if config[:encryption].nil?
         true
       else
-        config[:pusher_encrypted]
+        config[:encryption]
       end
     end
 
+    def debug_flag
+      return config[:debug] unless config.fetch(:debug, nil).nil?
+      defined?(::Rails) && !::Rails.env.production?
```

Comment:
Care to explain this a bit? I don't think we're setting any configs automatically based on the Rails environment, so it might be best to just leave this as:

```rb
def debug_flag
  config[:debug]
end
```

...unless I'm missing something?